lib/network.go

```diff
@@ -2346,6 +2361,13 @@ func DecodeHeaderVersion2(rr io.Reader) (*MsgDeSoHeader, error) {
 	retHeader.Nonce = 0
 	retHeader.ExtraNonce = 0
 
+	// TxnConnectStatusByIndexHash
+	retHeader.TxnConnectStatusByIndexHash = &BlockHash{}
```

Comment:
do we need some sort of new migration or header version to handle this?

Reply:
No migration needed here. The header version above defines whether the field exists or not